### DISCUSSION TITLE
Exporting SNVs and INDELs

### DISCUSSION THREAD

**Discussion by LegebekeJe** [Clinical-Genomics/scout#3814] [Q&A] [answered]
When I look into my SNV and INDELs list there are 2540/2774 filtered variants. However, when I click ' filter and export' I only get 500 variants in an Excel file. Where is the rest?

Also for this example ignoring the indels but there are roughly 4 - 5 million SNPs in a person's genome. What kind of filtering is performed in SCOUT to reduce this humongous number down to roughly a few thousand?

**Comment by northwestwitch** [comment]
>When I look into my SNV and INDELs list there are 2540/2774 filtered variants. However, when I click ' filter and export' I only get 500 variants in an Excel file. Where is the rest?

Hi @LegebekeJe, the variant export returning only 500 variants is designed like this on purpose: scout performs all the computation and returns responses in real time. It's a process under the hood, but for each variant that is exported there are queries runned on the database in order to "decorate" the returned variant info with more data, for instance genes, transcripts etc. 

We haven't really tested the limit, but I'm quite sure that exporting thousands of variants will cause the responses to hang. 

The idea is that you first filter your variants of interest using the filters provided on the variantS page, and then you export the interesting ones with that button. Also because if you export thousands of variants you might as well analyse the VCF file, I guess?

> Also for this example ignoring the indels but there are roughly 4 - 5 million SNPs in a person's genome. What kind of filtering is performed in SCOUT to reduce this humongous number down to roughly a few thousand?

Scout is not filtering your variants, it's just showing the variants present in your VCF file. You can either filter your VCF before loading the variants into scout or use a scoring system so that your variants will be ranked when they are loaded. This way only those with score above a certain threshold will be loaded (the score threshold can also manually set when loading variants).

Here's the documentation page for ranking scores: https://clinical-genomics.github.io/scout/admin-guide/annotations/#rank-score

Hope this was helpful!

**Comment by LegebekeJe** [comment]
Thank you for your reply @northwestwitch. I indeed would like to try analyzing the raw VCF file myself, how can I obtain these?

**Comment by dnil** [comment, marked as answer]
Hi Jelmer! I think you should ask for a facility introduction from someone at Clinical Genomics e.g. via a support system ticket (https://clinical.scilifelab.se). Briefly you specify the mode of desired delivery in the Orderportal when you place your sequencing order. One of the options allows you to select delivery to a download server (caesar / clinical-delivery) and the types of files/pipeline you want. If you want to do that post hoc, the friendly and hard working production bioinformatics team will likely be able to help you via a similar support ticket as well. Bear in mind that if many people have requests like that, their time will be stretched thin.

We would like to think Scout is very well geared for clinical rare disease (and cancer) analysis, and to some degree rare disease research - but you may want to have a chat with someone here when planning so that you get the best setup for your cohort. If you are missing some features that you would like to display in your own analysis of such cases, it would be super helpful for us and the rest of the community if you request them on this forum and we can try to integrate them!

**Reply by dnil** [reply]
I converted this to a discussion instead of an issue thread: would be fun to hear what you plan to do with the data, and what other analysis you would apply!

**Comment by LegebekeJe** [comment]
@dnil Thank you for you reply. Good idea to request an introduction from someone at Clinical Genomics, I will raise a support system ticket. I am more than happy to come by.

I do agree that Scout is good platform for clinical genetic diagnostic analyses. However, I myself am involved in rare disease research and I find that Scout is either lacking some functions, or it is a bit cumbersome to assess multiple variants at once. I will write my suggestions/ideas/issues down so that I can ask them first during the Clinical Genomics introduction before spamming this forum.
